docs/environment_variables.md

```diff
@@ -0,0 +1,47 @@
+Environment Variables
+=====================
+
+The HTTPX library can be configured via environment variables.
+Here are a list of environment variables that HTTPX recognizes
+and what function they serve:
+
+HTTPX_DEBUG
+-----------
+
+Valid values: `1`, `true`
+
+If this environment variable is set to `1` or `true` then
+logging will be turned on by default to stderr about low-level
+details of the HTTP request and responses being sent and received.
+
+This can help you debug issues and see what's exactly being sent
+over the wire and to which location.
+
+Example:
+
+```python
+# test_script.py
+
+import httpx
+client = httpx.Client()
+client.request("GET", "https://google.com")
```

Comment:
I think we showcase `.get()` in other examples in the docs. Since `.request()` is not the focus here how about using `.get()` here as well for consistency?

```suggestion
client.get("https://google.com")
```